docs/Lua API.rst

```diff
@@ -4415,12 +4415,29 @@ General script API
   command-line. The ``name`` argument should be the name of the script without
   its extension, as would be used on the command line.
 
-  Note that this function lets Lua errors propagate to the caller.
+  Example::
+     dfhack.run_script("armoks-blessing", "all")
+  Scipts that use a dash(``-``) in their arguments still require one when called
+  from ``dfhack.run_script``.
+  Example::
+     dfhack.run_script("gaydar", "-all")
+
+  Unlike ``dfhack.run_command``, ``dfhack.run_script`` allows a different
+  different script to be executed and its value passed as an argument to the main
+  function called by ``dfhack.run_script``.
```

Comment:
I'm not sure what this is supposed to mean - could you clarify? `dfhack.run_command("armoks-blessing")` and `dfhack.run_script("armoks-blessing")` should do essentially the same thing. The main differences I know of are that `run_script` propagates exceptions (already mentioned) and that `run_command` returns a command_result (mentioned in the `run_command` docs).

Typo: "different" is duplicated.